app/src/main/java/com/example/poste/activities/LoginActivity.java

```diff
@@ -108,9 +108,13 @@ public void onResponse(Call<ResponseBody> call, Response<ResponseBody> response)
                         } catch (JSONException err){
                             Log.e("Json Error", err.toString());
                         }
-                        // open dashboard activity
+                        // open dashboard activity (CURRENTLY TAKES BACK TO INTRO, TAKING TO DASHBOARD CRASHES)
                         Intent dashboardIntent = new Intent(LoginActivity.this, IntroActivity.class);
                         finish();
+                        // The updateFoldersandPost should be called in Dashboard
```

Comment:
I think I might recommend uncommenting these lines and, if necessary, allowing the app to crash (preferably with logging) so we can determine why it crashes and how to resolve. Otherwise this sort of obfuscates the issue and control flow.

Not expecting you to debug / solve this in THIS pull request, but I'm weary of introducing unrelated control flow changes

Reply:
Sounds good. I will uncomment and let it crash then

Reply:
Best practice might be to create a user story to resolve, including specifics on this issue and what's causing it. That backlog of bugs will help us in this sprint.